src/server.coffee

```diff
@@ -286,3 +335,45 @@ if not module.parent
     process.on 'SIGINT', -> stop process.exit
     # terminate signal
     process.on 'SIGTERM', -> stop process.exit
+
+
+#############################################
+###   Restart the server - Agenda Job     ###
+#############################################
+
+exports.getCertKeyStatus = (callback) ->
```

Comment:
Perhaps we should actually move this to the api/keystore.coffee script. I think it would fit better there.